src/main/java/com/exacttarget/fuelsdk/ETClient.java

```diff
@@ -263,7 +263,10 @@ public synchronized String requestToken(String refreshToken)
         jsonObject.addProperty("clientId", clientId);
         jsonObject.addProperty("clientSecret", clientSecret);
         jsonObject.addProperty("accessType", "offline");
-        if (refreshToken != null) {
+        
+        if (configuration.isFalse("useRefreshToken")) {
```

Comment:
@phoebus58 because `isTrue` will return false if `"useRefreshToken"` is not set. We're trying to preserve the existing behavior.